tests/dynamic_data/publications/layer_external_db/external_db_test.py

```diff
@@ -175,6 +175,24 @@
         'exp_languages': set(),
         'exp_scale_denominator': None,  # Layman doesn't guess scale denominator for (multi)points
     },
+    'point_epsg_3034': {
+        'file_path': "tests/dynamic_data/publications/crs/vectors/sample_point_cz_3034.shp",
+        'style_file': 'tests/dynamic_data/publications/crs/vectors/sample_point_cz.sld',
+        'schema_name': 'public',
+        'table_name': 'point_3034',
+        'primary_key_column': 'ogc_fid',
+        'geo_column_name': 'wkb_geometry',
+        'db_connection_str': None,
+        'additional_geo_column': None,
+        'exp_thumbnail': 'tests/dynamic_data/publications/crs/vectors/sample_point_cz_3034_thumbnail.png',
+        'exp_thumbnail_max_diff': 300,
```

Comment:
Why 300? And why 500 in tests/dynamic_data/publications/crs/vectors/__init__.py ?

Reply:
300 because the thumbnail was generated for QML, not for SLD.

500 because of the same reason. It should be enough to detect bigger shifts of the blue circles.